### PR TITLE
SISRP-16880 EDO DB knows no Booleans

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.berkeley.edu/browse/SISRP-16880

### CODE REVIEW

File: app/models/edo_oracle/user_courses/base.rb [hunk 1]
```diff
@@ -119,17 +119,17 @@ def row_to_section_data(row, cross_listing_tracker=nil)
         section_data = {
           ccn: row['section_id'].to_s,
           instruction_format: row['instruction_format'],
-          is_primary_section: row['primary'],
+          is_primary_section: (row['primary'] == 'true'),
```
Comment (johncrossman) [inline app/models/edo_oracle/user_courses/base.rb]: @paulkerschen : how about a concise util. E.g., `to_b(string)`?  (No booleans in the db brings back memories of Matterhorn. Ouch.)

**Comment by paulkerschen** [comment]
Thanks, @johncrossman, util method seems warranted.

**Comment by paulkerschen** [comment]
Amended to use a utility method, and to rely on the stored boolean instead of converting over and over.

File: app/models/edo_oracle/user_courses/base.rb [hunk 2]
```diff
@@ -166,6 +169,10 @@ def merge_detailed_section_data(campus_classes)
         end
       end
 
+      def to_boolean(string)
```
Comment (redconfetti) [inline app/models/edo_oracle/user_courses/base.rb]: @paulkerschen `true` dat!

**Pull request merged**